schema/2.0.0/corpora_schema.md

```diff
@@ -0,0 +1,264 @@
+| is_primary_data | `Bool`. This MUST be `True` if this is the canonical instance of this cellular observation and `False` if not. This is commonly `False` for meta-analyses reusing data or for secondary views of data. | Curator |
+| organism_ontology_term_id | `str` or categorical with `str` categories. This MUST be either "NCBITaxon:9606" or "NCBITaxon:10090". | Curator |
+| sex_ontology_term_id | `str` or categorical with `str` categories. This MUST be a child of [PATO:0001894](http://purl.obolibrary.org/obo/PATO_0001894) | Curator |
+| tissue_ontology_term_id | `str` or categorical with `str` categories. This MUST be an UBERON term. This SHOULD be appended with " (cell culture)" or " (organoid)" if appropriate. | Curator |
+| | | |
+
+Curators SHOULD NOT annotate the following columns in the `obs` dataframe which are the human-readable names that MUST match the corresponding ontology term identifier. These columns MUST be automatically annotated by the cellxgene Data Portal when a dataset is uploaded.
+
+| Key | Value | Annotator |
+:--|:--|:--
+| assay | `str` or categorical with `str` categories. This MUST be appended with any *clarifying text* set in matching identifier. | Data Portal |
+| cell_type | `str` or categorical with `str` categories. | Data Portal |
+| development_stage | `str` or categorical with `str` categories. This MUST be "unknown" if set in the matching identifier | Data Portal |
+| disease | `str` or categorical with `str` categories | Data Portal |
+| ethnicity | `str` or categorical with `str` categories. This MUST be "na" or "unknown" if set in the matching identifier. | Data Portal |
+| organism | `str` or categorical with `str` categories | Data Portal |
+| sex | `str` or categorical with `str` categories | Data Portal |
+| tissue | `str` or categorical with `str` categories. This MUST be appended with " (cell culture)" or " (organoid)" if set in the matching identifier. | Data Portal |
+| | | |
+
+## `var` (Gene Metadata)
+
+`var` is a [`pandas.DataFrame`](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.html).
+
+`var.index` MUST contain unique ENSEMBL gene identifiers for features. 
+
+Curators MUST annotate the following columns in the `var` dataframe:
+
+| Key | Value | Annotator |
+:--|:--|:--
+| feature_biotype | This MUST be "gene". | Data Portal |
+| feature_id (`var.index`) | `str`. This MUST be an ENSEMBL term from the gene annotation corresponding to the organism for the gene.  | Curator |
+||||
+
+Curators SHOULD NOT annotate the following column in the `var` dataframe which is the human-readable name that MUST match the corresponding gene identifier. This column MUST be automatically annotated by the cellxgene Data Portal when a dataset is uploaded.
```

Comment:
```suggestion
When a dataset is uploaded, the cellxgene Data Portal will automatically add to the `var` dataframe the following human-readable names that MUST match the corresponding gene identifier. Curators SHOULD NOT annotate the following columns.
```

Same restructuring recommendation as in `obs`.

Reply:
Rewrote a bit. Addressed in https://github.com/chanzuckerberg/single-cell-curation/pull/67/commits/a0909191d6f636f2c857ece47eae5537d1ef9223

Reply:
I like it.